examples/arena.q

```diff
@@ -18,26 +18,26 @@
 \d .protobufkdb
```

Comment:
Let's remove this line as now namespace is added to function names. I think it is fine if functions are called with full domain in examples.

Reply:
done